agent/src/main/java/com/teamscale/jacoco/agent/store/upload/azure/AzureFileStorageUploadStore.java

```diff
@@ -176,24 +176,25 @@ private String createFileName() {
 	}
 
 	/** Creates and fills a file with the given data and name. */
-	private Response<ResponseBody> createAndFillFile(byte[] zipFilBytes, String fileName) throws UploadStoreException, IOException {
-		Response<ResponseBody> response = createFile(zipFilBytes, fileName);
+	private Response<ResponseBody> createAndFillFile(File zipFile,
+													 String fileName) throws UploaderException, IOException {
+		Response<ResponseBody> response = createFile(zipFile, fileName);
 		if (response.isSuccessful()) {
-			return fillFile(zipFilBytes, fileName);
+			return fillFile(zipFile, fileName);
 		}
 		logger.warn(String.format("Creation of file '%s' was unsuccessful.", fileName));
```

Comment:
why is this a warning and not an error? sounds like the upload fails in this case

Reply:
No clue, didn't write that line 🤷‍♂️Will change it to error, though.